INTITA/Modules/ProfileSettings/Cells/HeaderSettingsTableViewCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  HeaderSettingsTableViewCell.swift
+//  INTITA
+//
+//  Created by Viacheslav Markov on 19.12.2020.
+//
+
+import UIKit
+
+protocol HeaderSettingsTableViewCellDelegate: AnyObject {
+    func editTaped(_ sender: HeaderSettingsTableViewCell)
+}
+
+class HeaderSettingsTableViewCell: UITableViewCell {
+    
+    var delegate: HeaderSettingsTableViewCellDelegate!
+    static let identifire = "HeaderSettingsTableViewCell"
+
+    @IBOutlet weak var mainView: UIView!
+    @IBOutlet var avatarView: UIImageView!
+    @IBOutlet var specializationLabel: UILabel!
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet weak var editButton: UIButton!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        setupCell()
+
+    }
+    
+    @IBAction func editButtonTapped(_ sender: UIButton) {
+        
+        delegate.editTaped(self)
+                
+    }
+    
+    static func nib() -> UINib {
+        return UINib(nibName: identifire, bundle: nil)
+    }
```

Comment:
`static func nib() -> UINib` and `static let identifire = "HeaderSettingsTableViewCell"`
were created twice in different classes. Maybe, it makes sense to take it out to protocol or extention.

Reply:
I'm agree on that